angular-ngrx-scss/src/app/file-viewer/components/file-explorer-about/file-explorer-about.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { RepoState } from '../../../state/repository';
+
+@Component({
+  selector: 'app-file-explorer-about',
+  templateUrl: './file-explorer-about.component.html',
+  styleUrls: ['./file-explorer-about.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FileExplorerAboutComponent {
+  @Input() repo!: RepoState | null;
+
+  get description(): string | null | undefined {
```

Comment:
Following the point above (the dumb component one), the data is gotten already by an ancestor (which does so using a selector), so all this component needs is for that to pass the data down to it.
The getters may be a slight overkill since I could simply rely on the 'repo' prop and use null checks in the template, however, I wanted a bit of consistency.